open-api/rest-catalog-open-api.yaml

```diff
@@ -3260,6 +3260,28 @@ components:
           additionalProperties:
             type: string
 
+    FineGrainedDataProtectionRules:
```

Comment:
How about simply `ProtectionPolicy`? It's a bit broad, but at the same time I'm not sure we necessarily need to include "FineGrained" in the name and I don't think we need to include "data" in the name either. The layer this structre is embedded at, along with its fields indicate the granularity already imo.

Reply:
Agree, i was refraining from using the term `Policy` becuase because it might get overloaded specially because catalog sees the raw polices, find applicable policies and then returns them here in this construct, how about using `ReadRestrictions` as ryan suggest or ProtectionInstructions

Reply:
Same here. `FineGrained` seems unnecessary.